core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/CoreTerminologyBroker.java

```diff
@@ -369,6 +393,16 @@ public boolean isEffectiveTimeSupported(final String terminologyId) {
 		throw new RuntimeException("Cannot find terminology with ID: '" + terminologyId + "'.");
 	}
 	
+	public SortedSet<String> getAffectedCodeSystemsForTeminology(final String terminologyId) {
+		Preconditions.checkNotNull(terminologyId, "terminologyId");
```

Comment:
Unnecessary null check. `Map.containsKey()` will return false and the `null` value will be reported by the `IllegalArgumentException`. I suggest using `Preconditions.checkArgument(boolean, String)`.